frontend/src/metabase/sharing/components/SlackChannelField.tsx

```diff
@@ -62,15 +62,17 @@ const SlackChannelField = ({
   };
 
   const applicationName = useSelector(getApplicationName);
+
   return (
     <div>
       <span className={cx(CS.block, CS.textBold, CS.pb2)}>
         {channelField?.displayName}
       </span>
-      <AutocompleteInput
-        placeholder={t`Pick a user or channel...`}
+      <Autocomplete
+        data={channelField?.options || []}
         value={value}
-        options={channelField?.options}
+        placeholder={t`Pick a user or channel...`}
+        limit={10}
```

Comment:
Does this mean it won't show more channels if users have more than 10? I played around in [the Mantine doc](https://v6.mantine.dev/core/autocomplete/#limit-amount-of-options), and I'm not sure this is our desired outcome